python-sdk/src/astro/files/locations/ftp.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+from urllib.parse import urlparse
+
+import smart_open
+from airflow.providers.ftp.hooks.ftp import FTPHook
+
+from astro.constants import FileLocation
+from astro.files.locations.base import BaseFileLocation
+
+
+class FTPLocation(BaseFileLocation):
+    """Handler FTP object store operations"""
+
+    location_type = FileLocation.FTP
+    supported_conn_type = {FTPHook.conn_type}
+
+    @property
+    def hook(self) -> FTPHook:
+        return FTPHook(ftp_conn_id=self.conn_id) if self.conn_id else FTPHook()
+
+    @property
+    def transport_params(self) -> dict:
+        """get FTP credentials for remote file system"""
```

Comment:
Are there no `transport_params` required for FTP?

Reply:
No not required its covered in the path itself with uri.